apps/ideas/views.py

```diff
@@ -32,28 +32,44 @@ def get(self, request, *args, **kwargs):
             'attachment; filename="ideas.csv"'
         )
 
-        # Selects all parent classes named ideas.models.abstracts.*Section
-        abstracts_module_name = abstracts.__name__ + '.'
-        abstract_sections = [
-            base_model for base_model in self.model.__mro__
-            if base_model.__module__.startswith(abstracts_module_name)
-            and base_model.__name__.endswith('Section')
-            ]
-
-        field_names = ['id']
-        for section in abstract_sections:
-            for field in section._meta.concrete_fields:
+        exclude_fields = ['module', 'item_ptr', 'members',
+                          'slug', 'idea_ptr', 'idea_image']
+
+        field_names = []
+        for field in IdeaSketch._meta.concrete_fields:
+            if (field.name not in exclude_fields
+                    and field.name not in field_names):
+                field_names.append(field.name)
+
+        for field in Proposal._meta.concrete_fields:
+            if (field.name not in exclude_fields
+                    and field.name not in field_names):
                 field_names.append(field.name)
+
         field_names.append('link')
         field_names.append('type')
 
-        writer = csv.writer(response)
+        writer = csv.writer(response, lineterminator='\n',
+                            quotechar='"', quoting=csv.QUOTE_ALL)
```

Comment:
I don't get a hundred percent what is happening here.

 - Isn't quotechar `"` the default setting anyway?
 - We set the lineterminators to unix, because windows lineterminators don't work with Exel?